open_rarity/scoring/handlers/information_content_scoring_handler.py

```diff
@@ -109,6 +115,9 @@ def _score_token(
 
         Returns:
             float: The token score
+
+        Limitations:
+            Does not take into account non-String attributes during scoring.
```

Comment:
aren't we failing the whole computation instead?

Reply:
We do it at the scorer level not handler level